docs/VerifPlans/Simulation/debug-trace/CV32E40X_debug.json

```diff
@@ -0,0 +1,1523 @@
+[
```

Comment:
The debug-trace folder contains two .json files:
1) CV32E40SX_debug.json
2) CV32E40X_debug.json
Is that correct? I would guess the CV32E40X_debug.json should be deleted?

Reply:
Whu... Good catch. Apparently an old version has laid doormant for 2 years.

I've deleted it now (the xlsx, plus csv/json).

![bilde](https://github.com/openhwgroup/cv32e40x-dv/assets/75431602/4a02cf72-f7bc-4f71-9710-c6221522a064)

Reply:
Aaa, I see. Good that it is fixed.